src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -57,7 +58,9 @@ namespace pluginhost {
 
         const std::vector<std::string> pluginPaths;
     };
-    RegistrationContext::RegistrationContext() : m_impl(new Impl) {}
+
+    RegistrationContext::RegistrationContext()
+        : m_impl(new Impl), m_logger(util::log::make_logger("PluginHost")) {}
```

Comment:
For things like this we should have a single-location-defined constant string `static const auto PLUGIN_HOST_LOGGER_NAME = "PluginHost";` lest our logs get split or something. (same with the osvr server app and maybe even osvrServer the library that also shares that same key - what's the technical term for that thingie now?)